overlord/hookstate/ctlcmd/refresh.go

```diff
@@ -234,6 +259,22 @@ func getUpdateDetails(context *hookstate.Context) (*updateDetails, error) {
 	return &up, nil
 }
 
+func (c *refreshCommand) printTrackingInfo(context *hookstate.Context) error {
+	context.Lock()
+	defer context.Unlock()
+
+	st := context.State()
+	var snapst snapstate.SnapState
+
+	err := snapstate.Get(st, context.InstanceName(), &snapst)
+	if err != nil {
+		return fmt.Errorf("internal error: cannot get snap state for %q: %v", context.InstanceName(), err)
```

Comment:
bit of nitpick but this error message will say the same thing twice since the "no state" error is already something like `no state entry for key "foo"`